scripts/RelayHubGasEstimation.ts

```diff
@@ -54,374 +63,429 @@ function logTitle(title: string) {
-  let verifier: TestVerifierEverythingAccepted;
-  let recipient: TestRecipient;
-  let token: UtilToken;
-  let owner: Wallet;
-  let forwarder: SmartWallet
-  let relayRequest: RelayRequest;
-
-  const deployAndSetup = async () => {
-    const provider = new ethers.providers.JsonRpcProvider(RSK_URL);
-  
-    const deployContract = <Contract>(contract: string) => {
-      return ethers
-        .getContractFactory(contract)
-        .then((contractFactory) => contractFactory.deploy() as Contract)
-    }
-  
-    owner = ethers.Wallet.createRandom().connect(provider)
-    penalizer = await deployContract<Penalizer>('Penalizer')
-    verifier = await deployContract<TestVerifierEverythingAccepted>(
-      'TestVerifierEverythingAccepted',
-    )
-    recipient = await deployContract<TestRecipient>('TestRecipient')
-    token = await deployContract<UtilToken>('UtilToken')
-    const smartWalletTemplate = await deployContract<SmartWallet>('SmartWallet')
-  
-    relayHub = await deployRelayHub(penalizer.address)
-  
-    await fundedAccount.sendTransaction({
-      to: owner.address,
-      value: ethers.utils.parseEther('1'),
-    })
-  
-    const factory = (await createSmartWalletFactory(
-      smartWalletTemplate,
-      false,
-      owner,
-    )) as SmartWalletFactory
-  
-    forwarder = await createSupportedSmartWallet({
-      relayHub: relayHubSigner.address,
-      factory,
-      owner,
-      sender: relayHubSigner,
-      logGas: true
-    }) as SmartWallet;
-  
-    await token.mint('1000', forwarder.address)
-  
-    relayRequest = {
-      request: {
-        relayHub: relayHub.address,
-        to: recipient.address,
-        data: '0xdeadbeef',
-        from: owner.address,
-        nonce: (await forwarder.nonce()).toString(),
-        value: '0',
-        gas: '3000000',
-        tokenContract: token.address,
-        tokenAmount: '1',
-        tokenGas: '50000',
-        validUntilTime: '0',
-      },
-      relayData: {
-        gasPrice,
-        feesReceiver: relayWorker.address,
-        callForwarder: forwarder.address,
-        callVerifier: verifier.address,
-      },
-    }
-  
-    await relayHub
-      .connect(relayOwner)
-      .stakeForAddress(relayManager.address, 1000, {
-        value: ethers.utils.parseEther('2'),
-      })
-  
-    await relayHub.connect(relayManager).addRelayWorkers([relayWorker.address])
-    await relayHub.connect(relayManager).registerRelayServer('http://relay.com')
-
-    return {
-      forwarder, recipient, relayHub, owner, relayRequest
-    }
-  }
+const deployContract = <Contract>(contract: string) => {
+  return ethers
+    .getContractFactory(contract)
+    .then((contractFactory) => contractFactory.deploy() as Contract);
+};
 
-  async function printGasStatus(receipt: ContractReceipt) {
-    const [firstAccount] = (await ethers.getSigners()) as [SignerWithAddress]
-    await token.mint('1000', firstAccount.address)
+async function setupRelayHub(relayHub: RelayHub) {
```

Comment:
Why do we define the functions with arrow functions and other as normal functions? I think we should try to be as much consistent as possible, so unless there's a specific reason, I'd apply always the same strategy to create functions.

Reply:
Done.